hoptimator-k8s/src/main/java/com/linkedin/hoptimator/k8s/K8sYamlApi.java

```diff
@@ -34,6 +34,19 @@ public void create(String yaml) throws SQLException {
     createWithAnnotationsAndLabels(yaml, null, null);
   }
 
+  // Returns the K8s Object or null if throwIfNotExists is set to false and the object does not exist.
+  public DynamicKubernetesObject get(String yaml, boolean throwIfNotExists) throws SQLException {
```

Comment:
boolean flags like this are hard to read; e.g. `get(x, false)` makes you go look at the docs to see what `false` does. Prefer using a different signature, e.g. `getIfExists(yaml)`.